src/cms/views/users/region_user_view.py

```diff
@@ -111,8 +111,8 @@ def post(self, request, *args, **kwargs):
                 user = region_user_form.save()
                 user_profile_form.save(user=user, region=region)
 
-                if user_profile_form.cleaned_data["send_activation_link"]:
-                    send_activation_link(request, user)
+                activation = user_profile_form.cleaned_data["send_activation_link"]
+                send_welcome_mail(request, user, activation)
```

Comment:
```suggestion
                    if not user_instance:
                        activation = user_profile_form.cleaned_data["send_activation_link"]
                        send_welcome_mail(request, user, activation)
```
Sorry, in my suggestion of my last review, I forgot that this would lead to the email being sent every time the form gets submitted. I suggest to insert a check whether the user instance already exists and only sent the mail if it doesn't.